testsite/bower.json

```diff
@@ -0,0 +1,22 @@
+{
+  "name": "testsite",
+  "version": "1.0.1",
+  "homepage": "https://github.com/counsyl/civet",
+  "authors": [
+    "opensource@counsyl.com"
+  ],
+  "description": "Civet test app",
+  "license": "Apache License 2.0",
+  "ignore": [
+    "**/.*",
+    "node_modules",
+    "bower_components",
+    "test",
+    "tests"
+  ],
+  "dependencies": {
+    "jquery": "~2.1.3",
+    "bootstrap-sass-official": "~3.3.1",
+    "test": "aristretto/test"
```

Comment:
This probably should be removed? I'm seeing this error with `bower install`:

```
bower test#*                                  ECMDERR Failed to execute "git ls-remote --tags --heads git://github.com/aristretto/test.git", exit code of #128

Additional error details:
fatal: remote error: 
  Repository not found.
```

https://github.com/aristretto/test does not exist it seems.